processor/winevent.go

```diff
@@ -0,0 +1,187 @@
+package processor
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/devopsext/events/common"
+	sreCommon "github.com/devopsext/sre/common"
+)
+
+type WinEventPubsubRequest struct {
+	Time     string                   `json:"time"`
+	Channel  string                   `json:"channel"`
+	Original *WinEventOriginalRequest `json:"data"`
+}
+type WinEventOriginalRequest struct {
+	Events []WinEvent `json:"metrics"`
+}
+type WinEventTags struct {
+	Message      string `json:"Message"`
+	MessageLevel string `json:"LevelText"`
+	Keywords     string `json:"Keywords"`
+	EventId      string `json:"EventRecordID"`
```

Comment:
should be EventID by go naming convention

Reply:
This should use golang notation. Agree